vue/sbc-common-components/src/components/SbcHeader.vue

```diff
@@ -464,8 +464,9 @@ export default class SbcHeader extends Mixins(NavigationMixin) {
 
   @Watch
   get alertColor(): string {
-    switch (this.environment) {
```

Comment:
this.environment?.toUpperCase()?

otherwise if this.environment is null, it will throw an exception

Reply:
nvm, we're good:

```
@Prop({ default: '' }) environment!: string;
```